EssentialsDiscord/src/main/java/net/essentialsx/discord/DiscordSettings.java

```diff
@@ -246,7 +246,7 @@ public MessageFormat getQuitFormat(Player player) {
             filled = format;
         }
         return generateMessageFormat(filled, ":arrow_left: {displayname} has left!", false,
-                "username", "displayname", "quitmessage");
+                "username", "displayname", "quitmessage", "online");
```

Comment:
Is there a reason there's no `"unique"` here?

Reply:
just small amounts of trolling, that's about it
```suggestion
                "username", "displayname", "quitmessage", "online", "unique");
```